package.json

```diff
@@ -15,7 +15,8 @@
     "lint": "tslint src/**/*.ts",
     "build": "yarn run clean && yarn run lint && tsc -p ./tsconfig.es2015.json && tsc -p ./tsconfig.cjs.json && node rollup.build.js",
     "prepare": "yarn run build",
-    "storybook": "start-storybook -p 9001 -c .storybook"
+    "storybook": "start-storybook -p 9001 -c .storybook",
+    "demo": "vite"
```

Comment:
What do we need vite for that storybook can't accomplish?

Reply:
Apparently storybook doesn't support Vue 3 yet.

Reply:
As James said, I only added vite to have some demo while we wait for storybook vue3 support.